content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,197 @@
+[4]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
+{{% /tab %}}
+
+{{% tab "Amazon MWAA" %}}
+## Requirements
+
+* [Apache Airflow 2.8.3][1] or later is required.
+* [apache-airflow-providers-openlineage 1.11.0][2] or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Install `openlineage` provider by adding the following into your `requirements.txt` file:
+   
+   ```text
+   apache-airflow-providers-openlineage>=1.11.0
+   ```
+
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables in your [Amazon MWAA start script][3]:
+   
+   ```shell
+   #!/bin/sh
+   
+   export OPENLINEAGE_URL=https://data-obs-intake.{{< region-param key="dd_site" code="true" >}}
+   export OPENLINEAGE_ENDPOINT=api/v1/lineage
+   export OPENLINEAGE_API_KEY=<DD_API_KEY>
+   ```
+
+   Check official documentation [configuration-openlineage][4] for other supported configurations of `openlineage` provider.
+
+   **Important:**
+   * replace `<DD_API_KEY>` fully with your valid [Datadog API key][5].
+
+3. Deploy your updated `requirements.txt` and [Amazon MWAA start script][3] to your Amazon S3 folder you configured for your Amazon MWAA Environment.
+
+4. Ensure your Execution role configured for your Amazon MWAA Environment has the right permissions to the `requirements.txt` and [Amazon MWAA start script][3]. This is required if you are managing your own Execution role and it's the first time you are adding those supporting files. See official guide [Amazon MWAA execution role][6] for details if needed.
+
+
+[1]: https://github.com/apache/airflow/releases/tag/2.7.0
+[2]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/index.html
+[3]: https://docs.aws.amazon.com/mwaa/latest/userguide/using-startup-script.html
+[4]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/configurations-ref.html#configuration-openlineage
+[5]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
+[6]: https://docs.aws.amazon.com/mwaa/latest/userguide/mwaa-create-role.html
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Validation
+
+In Datadog, view the [Data Jobs Monitoring][2] page to see a list of all your Airflow jobs.
+
+## Advanced Configuration
+
+### Link your Spark jobs with Airflow task
+To allow your Airflow job linked to the the Spark application it submitted, following the following steps:
+
+1. Configure your Airflow to turn off lazily loading Airflow plugins by setting [lazy_load_plugins config][3] to `False` in your `airflow.cfg` or exporting the following environment variable where your Airflow schedulers and Airflow workers run:
+   
+   ```shell
+   export AIRFLOW__CORE__LAZY_LOAD_PLUGINS='False' 
```

Comment:
Is the `LAZY_LOAD_PLUGINS` only needed for the macros, or also for the regular openlineage capabilities ?

Reply:
it's just for the macros.